light-curve/tests/light_curve_py/test_single_band.py

```diff
@@ -0,0 +1,75 @@
+import numpy as np
+import pytest
+
+from light_curve.light_curve_py import LinearFit
+
+
+def test_no_init_bands_no_input_bands():
+    """Announce no bands, use no bands"""
+    t = [1.0, 2.0, 3.0, 4.0, 5.0]
+    m = [1.0, 2.0, 3.0, 4.0, 5.0]
+    sigma = [1.0, 1.0, 1.0, 1.0, 1.0]
+    feature = LinearFit(bands=None)
+    _values = feature(t, m, sigma)
+
+
+def test_init_bands_no_input_bands():
+    """Announce bands, use no bands"""
+    t = [1.0, 2.0, 3.0, 4.0, 5.0]
+    m = [1.0, 2.0, 3.0, 4.0, 5.0]
+    sigma = [1.0, 1.0, 1.0, 1.0, 1.0]
+    feature = LinearFit(bands=["g"])
+    with pytest.raises(ValueError):
+        _values = feature(t, m, sigma)
+
+
+def test_no_init_bands_input_bands():
+    """Announce no bands, use input bands"""
+    t = [1.0, 2.0, 3.0, 4.0, 5.0]
+    m = [1.0, 2.0, 3.0, 4.0, 5.0]
+    sigma = [1.0, 1.0, 1.0, 1.0, 1.0]
+    bands = ["g", "g", "g", "r", "r"]
+    feature = LinearFit(bands=None)
+    with pytest.raises(ValueError):
+        _values = feature(t, m, sigma, bands)
+
+
+def test_init_bands_eq_input_bands():
+    """Announce no bands, use input bands"""
+    t = [1.0, 2.0, 3.0, 4.0, 5.0, 6.0]
+    m = [1.0, 2.0, 3.0, 4.0, 5.0, 6.0]
+    sigma = [1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
+    bands = ["g", "g", "g", "r", "r", "r"]
+    feature = LinearFit(bands=["g", "r"])
+    values = feature(t, m, sigma, bands)
+    assert 2 == feature.n_bands
+    assert values.size == feature.size
+    assert values.size == feature.size_single_band * feature.n_bands
+
+
+def test_init_bands_less_input_bands():
+    t = [1.0, 2.0, 3.0, 4.0, 5.0, 6.0]
+    m = [1.0, 2.0, 3.0, 4.0, 5.0, 6.0]
+    sigma = [1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
+    bands = ["g", "g", "g", "r", "r", "r"]
+    feature = LinearFit(bands=["g"])
+    values = feature(t, m, sigma, bands)
+    assert 1 == feature.n_bands
+    assert values.size == feature.size
+    assert values.size == feature.size_single_band
+
```

Comment:
Probably, we also want to test the case when bands in feature constructor is inconsistent with input bands? 

Something like:

```
def test_inconsistent_bands():
     """Announced bands and input bands are different"""
     t = [1.0, 2.0, 3.0, 4.0, 5.0]
     m = [1.0, 2.0, 3.0, 4.0, 5.0]
     sigma = [1.0, 1.0, 1.0, 1.0, 1.0]
     bands = ["g", "g", "g", "g", "g"]
     feature = LinearFit(bands=["r"])
     with pytest.raises(ValueError):
         _values = feature(t, m, sigma, bands)
```

Or we check it with the next test?

Reply:
Thank you, more tests is always better!